R/data.R

```diff
@@ -1,5 +1,52 @@
 #' WokeWindows citations
 #' @docType data
+#' @format
+#' An object of class \code{tbl_df} (inherits from \code{tbl}, \code{data.frame}) with 328141 rows and 45 columns.
+#' \describe{
+#'   \item{issuing_agency}{Boston Police division that issuing officer belongs to. a character vector.}
+#'   \item{agency_code}{Shorthand code for issuing agency. a character vector.}
+#'   \item{officer_id}{Unique identifier for BPD offficers. a character vector.}
+#'   \item{officer_name}{The name of the issuing officer.}
+#'   \item{event_date}{Year-Month-Day.}
+#'   \item{time_hh}{Hour of incident.}
```

Comment:
What do you think about converting these four variables into a timestamp? Wouldn't that be more efficient storage-wise?